packages/contracts/arguments.js

```diff
@@ -0,0 +1,6 @@
+module.exports = [
+  // Demo url
+  "https://offchain-resolver-example.uc.r.appspot.com/{sender}/{data}.json",
```

Comment:
I think these should probably be set as variables in hardhat.config.js - you can then access them from the network object in a deployment.

Reply:
`arguments.js` is a stand alone file which gets specified when running the command line so I don't think argument.js can read the environment variable from hardhat.config.js (such feature is not specified at https://hardhat.org/plugins/nomiclabs-hardhat-etherscan.html either)

```
npx hardhat verify --constructor-args arguments.js DEPLOYED_CONTRACT_ADDRESS

```

Reply:
You can set custom variables in the 'network' object for each network, and then access them from the network object inside the deployment.